include/wx/msw/listctrl.h

```diff
@@ -154,6 +154,9 @@ class WXDLLIMPEXP_CORE wxListCtrl: public wxListCtrlBase
     // Gets information about the item
     bool GetItem(wxListItem& info) const;
 
+    // Check if the item is visible
+    bool IsVisible(long item);
```

Comment:
```suggestion
    bool IsVisible(long item) wxOVERRIDE;
```

Reply:
@MaartenBent ,
This will also probably won't be needed.

Reply:
Should get `const wxOVERRIDE`, see final comment.